Documentation/building/unix-instructions.md

```diff
@@ -59,3 +59,19 @@ If you see errors along the lines of `SendFailure (Error writing headers)` you m
 ```sh
 mozroots --import --sync
 ```
+
+## Steps to Build on Ubuntu
+
+*Note: verified on Ubuntu 14.04 LTS*
+
+1. Install git `sudo apt-get install git`
+2. Clone the corefx repo `git clone https://github.com/dotnet/corefx.git`
+3. Install curl-dev `sudo apt-get install libcurl4-openssl-dev`
```

Comment:
One last thing: could you remove the `sudo apt-get ...` from the bulleted section? Not all of the platforms we support use apt-get, and I think the "copy-paste command" in the Ubuntu14.04 section should be good enough for folks who want an easy route to install everything on a platform with apt-get.